From 5ac06eb74cb7f4a72914fd842d53ae90a1dc5319 Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Tue, 5 May 2026 13:32:37 +0800
Subject: [PATCH] ram: aspeed: add SDRAM support for AST2705

AST2705 reuses the AST2700 SDRAM driver, so wire CONFIG_ASPEED_AST2705
into the RAM Makefile.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Iccc71e72499842f49f49a5bccb964a7c7f5a325c
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 drivers/ram/aspeed/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/ram/aspeed/Makefile b/drivers/ram/aspeed/Makefile
index 868194c7bb..3107a908c3 100644
--- a/drivers/ram/aspeed/Makefile
+++ b/drivers/ram/aspeed/Makefile
@@ -4,4 +4,5 @@ obj-$(CONFIG_ASPEED_AST2500) += sdram_ast2500.o
 obj-$(CONFIG_ASPEED_AST2600) += sdram_ast2600.o
 obj-$(CONFIG_TARGET_ASPEED_AST2700) += sdram_ast2700.o sdram_phy.o
 obj-$(CONFIG_ASPEED_AST2700) += sdram_ast2700.o
+obj-$(CONFIG_ASPEED_AST2705) += sdram_ast2700.o
 obj-$(CONFIG_ASPEED_FPGA) += fpga_phy.o
-- 
2.43.0

